spec/unit/throttler/slave_lag_spec.rb

```diff
@@ -206,6 +206,7 @@ def @throttler.max_current_slave_lag
 
     describe 'with MySQL stopped on the slave' do
       it 'assumes 0 slave lag' do
+        skip "Failing test that needs review"
```

Comment:
Backtrace:
```
NoMethodError: undefined method `exclude?' for []:Array
    /Users/brianb/src/github.com/Shopify/lhm/lib/lhm/throttler/slave_lag.rb:65:in `get_slaves'
    /Users/brianb/src/github.com/Shopify/lhm/lib/lhm/throttler/slave_lag.rb:55:in `slaves'
    /Users/brianb/src/github.com/Shopify/lhm/lib/lhm/throttler/slave_lag.rb:82:in `max_current_slave_lag'
    /Users/brianb/src/github.com/Shopify/lhm/spec/unit/throttler/slave_lag_spec.rb:217:in `block (4 levels) in <top (required)>'
```

Original commit: https://github.com/Shopify/lhm/commit/a0df0e922247dce7aa10fa8f5c3c3d791d630ef2

Reply:
So, I think we have a hidden dependency on ActiveSupport?

https://github.com/rails/rails/blob/379d98dcd40f28fd959f42958d662b5416171eea/activesupport/lib/active_support/core_ext/enumerable.rb#L102

We could always make the dependency explicit, given that LHM is usually only ran in a Rails context.